src/infinitecanonicalmps.jl

```diff
@@ -48,11 +60,82 @@ function shift_flux(i::Index, flux_density::QN)
   return ITensors.setspace(i, shift_flux(space(i), flux_density))
 end
 
+function multiply_flux(qn::ITensors.QNVal, flux_factor::Int64)
```

Comment:
I would prefer the name `scale_flux`. Also maybe allow more general `flux_factor` inputs with `::Integer`.